src/transport/SessionManager.cpp

```diff
@@ -551,6 +551,12 @@ void SessionManager::OnMessageReceived(const PeerAddress & peerAddress, System::
         return;
     }
 
+    if (packetHeader.HasPrivacyFlag())
```

Comment:
What is the path for rolling this out, given that some implementations are shipping something around sve2, not 1.0, and hence are still setting the P flag?